WolvenKit.Modkit/Scripting/ScriptFunctions.cs

```diff
@@ -188,8 +189,11 @@ public virtual IEnumerable GetArchiveFiles()
     /// <returns>The converted JSON string</returns>
     public virtual string YamlToJson(string yamlText)
     {
-        var deserializer = new Deserializer();
-        var yamlObject = deserializer.Deserialize(yamlText);
+        // remove yaml tags like !include, !append etc.
+        yamlText = YamlTagRegex().Replace(yamlText, "");
```

Comment:
Why do you strip the tags here?

Reply:
because the yaml doesn't compile while they're present :| 